src/Locale.php

```diff
@@ -35,7 +37,7 @@ final class Locale implements MiddlewareInterface
     private const DEFAULT_LOCALE_NAME = '_language';
     private const LOCALE_SEPARATORS = ['-', '_'];
 
-    private bool $saveLocale = true;
+    private bool $saveLocale = false;
```

Comment:
Since we now can turn on/off session saving and cookie saving, can we remove general `withSaveLocale()` option?

Reply:
https://github.com/yiisoft/yii-middleware/issues/83